src/components/Layout/Header/NavBar/Notifications.tsx

```diff
@@ -8,15 +11,34 @@ import {
 } from '@wherever/react-notification-feed'
 import { getConfig } from 'config'
 import { useMemo } from 'react'
+import { useSelector } from 'react-redux'
+import { KeyManager } from 'context/WalletProvider/KeyManager'
 import { getLocalWalletType } from 'context/WalletProvider/local-wallet'
-import { isKeyManagerWithProvider } from 'context/WalletProvider/WalletProvider'
 import { useFeatureFlag } from 'hooks/useFeatureFlag/useFeatureFlag'
+import { useWallet } from 'hooks/useWallet/useWallet'
+import { selectWalletAccountIds } from 'state/slices/common-selectors'
 import { breakpoints, theme } from 'theme/theme'
 
+const eip712SupportedWallets = [KeyManager.KeepKey, KeyManager.Native]
+
 export const Notifications = () => {
   const isWhereverEnabled = useFeatureFlag('Wherever')
   const { colorMode } = useColorMode()
+  const {
+    state: { wallet },
+  } = useWallet()
+
+  const walletAccountIds = useSelector(selectWalletAccountIds)
+
+  const ethFullAccount = walletAccountIds.find(accountId => {
+    const fullAccount = fromAccountId(accountId)
+    return fullAccount.chainId === 'eip155:1'
+  })
+
+  const ethAddress = ethFullAccount ? fromAccountId(ethFullAccount).account : ''
```

Comment:
i see what you're trying to do here, grab the `0x...` address from the full account id.

native and keepkey wallets both support an indefinite number of accounts, so this logic is just going to give back an eth address, but no guarantee of which account number (ordering is not guaranteed).

this is more of a product question, of how we want to support multiple accounts via wherever

Reply:
we have a component we use for this throughout the app https://github.com/shapeshift/web/blob/develop/src/components/AccountDropdown/AccountDropdown.tsx#L80

Reply:
The way we support multiple accounts should be the same way as we currently support multiple accounts in Wherever for Metamask I believe - the notifications the user is currently viewing should be for the current account the user has selected/active.
Not clear from the `AccountDropdown` how to select the activeAccount. Seems that the dropdown only stores the selected account in local state. Is there any way to understand globally which account is currently selected?

Reply:
Ok got it, thats what I was missing. I think that means we should consult with @DiggyDiggy2 then on how to proceed here.

Reply:
Looking at the UI it seems that when I create an SS wallet I get an account on every chain, but there is only 1 ETH account, and I don't have the option to add more or delete. If a Native / KeepKey user can only have 1 ETH account per wallet then we also shouldn't be have an issue by just automatically selecting the ETH account and using that to always display notifications

Reply:
@nivo33 Accounts are automatically added if there's a balance / Tx history on your next account, and we stop adding them as soon as an account without any balance/Tx history is found.

I suppose you're testing with empty account 0 and consequently 0+: alternatively, if yourwant to add the next account from your highest added account (0 -> 1 in your case), you can do so for any chain, including Ethereum, by navigating to the accounts page (`/accounts`) and clicking "Add Account":

<img width="1309" alt="image" src="https://user-images.githubusercontent.com/17035424/214280967-d4c06bae-8e4a-46d8-a045-648b9d7ff9bf.png">

Note that due to the way multi-account works, a Tx history on the currently highest account (0 in your case) is required to be able to add the next account (1 here) so if you want to test Wherever with multi-account, you'll either need to 

- do a small dust send on your default account 0 or 
- import a seed with a non empty 0th account from a wallet that uses the same standard BIP44 `account` incrementation logic (`m/44'/60'/0'/0/0` -> `m/44'/60'/1'/0/0` e.g Ledger Live, Bitfrost, but not e.g MetaMask, Gridplus, Trezor which increment `address_index` instead of `account`)

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/17035424/214281207-fe0f50cf-c058-4d78-98d6-91fc7db1c216.png">

Reply:
@gomesalexandre Got it, thanks! But account 0 will always be available right? There is no way to delete it?

Reply:
@nivo33 correct - there will always be account 0 for eth